<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1">
   <item row="1" column="0">
    <widget class="QGroupBox" name="plot_groupBox">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="show_pushButton">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="control_groupBox">
     <property name="title">
      <string>Camera Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="4">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="6">
         <widget class="QCheckBox" name="live_checkBox">
          <property name="text">
           <string>Live</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QDoubleSpinBox" name="framerate_doubleSpinBox"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Camera #</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QDoubleSpinBox" name="exp_doubleSpinBox"/>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="auto_exposure_comboBox"/>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Frame Rate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QCheckBox" name="auto_level_checkBox">
          <property name="text">
           <string>Auto Level display</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Exposure</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="cam_connect_checkBox">
          <property name="text">
           <string>HW Connect</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QDoubleSpinBox" name="cam_index_doubleSpinBox"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Auto Exposure:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QCheckBox" name="crosshairs_checkBox">
          <property name="text">
           <string>Crosshairs</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QPushButton" name="save_pushButton">
          <property name="text">
           <string>Save Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
